### ISSUE TITLE
[$250] [A11y] iOS - Chat - Unable to tap on link in the message when VoiceOver is enabled

### ISSUE THREAD

**Issue by jponikarchuk** [Expensify/App#80203]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.3.6-0
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**If this was caught during regression testing, add the test name, ID and link from BrowserStack:** https://github.com/Expensify/App/pull/79147
**Email or phone of affected tester (no customers):** soidjnosidjsojidojs@gmail.com
**Issue reported by:** Applause Internal Team
**Bug source:** Pull Request QA execution
**Device used:** iPhone 15 Pro Max / iOS 26.1
**App Component:** Chat Report View

## Action Performed:
Precondition:
- VoiceOver is enabled.

1. Launch Expensify app.
2. Open any chat with link in the message.
3. Try to tap on the link in the message.

## Expected Result:
User can navigate to the link in the message when VoiceOver is enabled.

## Actual Result:
User cannot reach to the link in the message when VoiceOver is enabled.

## Workaround:
Unknown

## Platforms:
- [ ] Android: App
- [ ] Android: mWeb Chrome
- [x] iOS: App
- [ ] iOS: mWeb Safari
- [ ] iOS: mWeb Chrome
- [ ] Windows: Chrome
- [ ] MacOS: Chrome / Safari

## Screenshots/Videos

https://github.com/user-attachments/assets/be2bbdd4-2946-44c3-a237-08cb448c0082

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~022014482690412987916</li>
        <li>Upwork Job ID: 2014482690412987916</li>
        <li>Last Price Increase: 2026-02-05</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @jliexpensify</details>

**Comment by jliexpensify** [comment]
@Pujan92 Hi, can you take over here? Cheers

**Comment by Pujan92** [comment]
I think we only need a change for ios(bcoz nested elements focus is an issue if accessible set for the parent element) as [suggested](https://github.com/Expensify/App/issues/80203#issuecomment-3841857668) by @Uzaifm127. 

🎀👀🎀 C+ reviewed

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by jliexpensify** [comment]
Have hired you @Uzaifm127, waiting for you to accept

**Comment by huangpei2026psq** [comment]
GitHub username: huangpei2026psq
I'm interested in this $250 bounty.
Please let me know the next steps. Thanks!

**Comment by Uzaifm127** [comment]
Thanks Jason, accepted the offer, requested the payment.
